README.md

```diff
@@ -1,57 +1,69 @@
-# 🚀 Getting started with Strapi
+# CoW CMS - Content Management System
 
-Strapi comes with a full featured [Command Line Interface](https://docs.strapi.io/dev-docs/cli) (CLI) which lets you scaffold and manage your project in seconds.
+This project is a Content Management System for the [cow.fi](https://cow.fi).
 
-### `develop`
+> It is an instance of Strapi, a headless CMS.
 
-Start your Strapi application with autoReload enabled. [Learn more](https://docs.strapi.io/dev-docs/cli#strapi-develop)
+# 👨‍💻 Develop
 
 ```
-npm run develop
-# or
-yarn develop
+yarn dev
 ```
 
-### `start`
+Some requiremets are:
 
-Start your Strapi application with autoReload disabled. [Learn more](https://docs.strapi.io/dev-docs/cli#strapi-start)
+- At least Node v16 (use a LTS version)
+- Yarn
+- PostgreSQL (only to run the Docker image), for local dev you can use sqlite3
 
-```
-npm run start
-# or
-yarn start
-```
+# Dev locally using PostreSQL
+
+The easiest is to develop using sqlite, but if you need to run the CMS with Docker, you can install an instance of PostgreSQL:
 
-### `build`
+- **Mac**: [Postgres.app](https://postgresapp.com/).
+- **Linux**: [PostgreSQL for Linux](https://www.postgresql.org/download/linux/)
+- **Windows**: [PostgreSQL for Windows](https://www.postgresql.org/download/windows/)
```

Comment:
This sentence and the instructions indicate that in order to use docker I need postgres installed?
Isn't it instead that we should run a postgres Docker img instead of installing it in the system?

Reply:
This PR was going to add docker already, but i will add in the next PR. 

I will make the docker things independent, so i will change this text. 
Docker is mostly useful for when we deploy, for our dev will be easier to just run `yarn dev`